### PR TITLE
Testing / documentation cleanup

### PR DESCRIPTION
**Pull request by tas50**
Align with modern best practices. Avoid managing deps in the Gemfile and just test with ChefDK

### CODE REVIEW

File: test/fixtures/cookbooks/test/recipes/default.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-execute 'apt-get update' if platform_family?('debian')
+execute 'apt-get update'
```
Comment (shortdudey123) [inline test/fixtures/cookbooks/test/recipes/default.rb]: Did you mean to remove the `if`?

Comment (tas50) [inline test/fixtures/cookbooks/test/recipes/default.rb]: Nope ripped through that too fast.

**Comment by tas50** [comment]
@flaccid This is step 1 before introducing integration testing with kitchen-dokken in Travis CI. Mostly just aligns things with other Sous Chefs and Chef Inc. cookbooks

**Comment by flaccid** [comment]
I certainly trust the judgement of the sous chefs! LGTM, thanks @tas50.

**Pull request merged**